### PR TITLE
Add images to questions

### PR DESCRIPTION
**Pull request by noahmalmed**
The swipe workflow didn't allow any images to show in the question. The pr allows for one image (the image from the question markdown) to be displayed alongside the 
# Review Checklist

- [x] Does it work in Android and iOS?
- [x] Can you `rm -rf node_modules/ && npm install` and app works as expected?
- [x] Are tests passing?

### CODE REVIEW

File: src/utils/markdownUtils.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+
+const imageRegEx = /(?: ?!\[[^\]]*\]\((https?:\/\/[^ ]*\.(?:png|jpg|gif|jpeg))[^\)]*\)|\n {0,3}\[[^\]]*]: *(https?:\/\/[^ ]*\.(?:png|jpg|gif|jpeg)))/i
+const markdownImageRegEx = / ?!\[[^\]]*\](\([^\)]*\)|\[[^\]]*\])/g
```
[the remark this answers is not in the document]
Comment (noahmalmed) [reply src/utils/markdownUtils.js]: @houndci-bot #hushpuppy

File: src/utils/markdownUtils.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
+
+const imageRegEx = /(?: ?!\[[^\]]*\]\((https?:\/\/[^ ]*\.(?:png|jpg|gif|jpeg))[^\)]*\)|\n {0,3}\[[^\]]*]: *(https?:\/\/[^ ]*\.(?:png|jpg|gif|jpeg)))/i
```
[the remark this answers is not in the document]
Comment (noahmalmed) [reply src/utils/markdownUtils.js]: @houndci-bot #hushpuppy

File: src/utils/markdownUtils.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (noahmalmed) [reply src/utils/markdownUtils.js]: I think we can ignore these, seems like hound is getting confused by the reg ex

File: src/utils/markdownUtils.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (noahmalmed) [reply src/utils/markdownUtils.js]: I think we can ignore these, seems like hound is getting confused by the reg ex

**Comment by wgranger** [comment]
Out of curiosity, what is the use case for this? Is there currently a project with an image in the question?

**Comment by noahmalmed** [comment]
Gravity spy is developing a workflow that is basically, swipe right if the image looks similar to the image in the question

**Comment by wgranger** [comment]
This looks nice. Code looks good and executes well. It's helpful this is in a utility, because I can see the same regex being used for images in answers, as is the case with Galaxy Zoo.

**Pull request merged**